internal/config/config_test.go

```diff
@@ -1,8 +1,11 @@
 package config
 
 import (
```

Comment:
This doesn't break anything, but don't some/most/all IDEs sort imports in the same order as `goimports`? (stdlib packages first, then public/third-party ones, etc)

Reply:
I am using `goimports` in GoLand which sort that way.

Reply:
linter was broken, I pushed a fix and needed to resolve various lint issues

Reply:
👏🏼